Makefile

```diff
@@ -47,6 +47,8 @@ endif
 
 SHELL = bash
 CXX ?= g++
+EMCC ?= emcc
+WASM_SHELL ?= d8
```

Comment:
Is there another standard option here? If so, might be worth a comment.

Reply:
Also might be worth mentioning that "wasm" is "WebAssembly" since both appear in this file and it is a good intro point for those not already in the know.